plugins/flatpak/gs-flatpak.c

```diff
@@ -1324,6 +1376,7 @@ gs_flatpak_add_updates (GsFlatpak *self, GsAppList *list,
 		const gchar *latest_commit;
 		g_autoptr(GsApp) app = NULL;
 		g_autoptr(GError) error_local = NULL;
+		GsApp *main_app = NULL;
```

Comment:
You're leaking `main_app` in this function

Reply:
Still applies